plugin/src/main/java/org/wildfly/swarm/plugin/gradle/GradleArtifactResolvingHelper.java

```diff
@@ -82,7 +82,7 @@ public ArtifactSpec resolve(final ArtifactSpec spec) {
     }
 
     @Override
-    public Set<ArtifactSpec> resolveAll(final Set<ArtifactSpec> specs, boolean transitive) throws Exception {
+    public Set<ArtifactSpec> resolveAll(final Set<ArtifactSpec> specs, boolean transitive, boolean defaultExcludes) throws Exception {
```

Comment:
I wanted to discuss and verify the general approach first before touching the other resolver implementations. But it's good you mention it, I'll make a note to look into it.

But didn't you mention on another thread that the gradle impl. should not be using the maven resolver, but some gradle specific components? Would that work similarly ?